src/CommunityVoices/Public/js/quote-form.js

```diff
@@ -12,3 +12,14 @@ function clickEdited() {
     originalText.value = '';
   }
 }
+
+function displayCount(box,count) {
+  var countDisplay = document.getElementById(count);
+  if(box.hasAttribute("suggestedlength") && (box.value.length > box.getAttribute("suggestedlength"))) {
+    countDisplay.innerHTML = box.value.length + " / " + box.getAttribute("maxlength") + " characters -- Please shorten!";
+    countDisplay.style.color = "red";
+  } else {
+    countDisplay.innerHTML = box.value.length + " / " + box.getAttribute("maxlength") + " characters";
```

Comment:
This line is repetitive.  I think that the `countDisplay.innerHTML` should be set to this line regardless and the addendum for suggested length should be added to it in a conditional.

Reply:
Always try to make your conditionals affect only the code that is truly different between the two.